src/main/java/com/hillel/items_exchange/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,48 @@
+package com.hillel.items_exchange.mapper;
+
+import com.hillel.items_exchange.dto.CategoryVo;
+import com.hillel.items_exchange.dto.SubcategoryVo;
+import com.hillel.items_exchange.model.Category;
+import com.hillel.items_exchange.model.Subcategory;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@RequiredArgsConstructor
+public class CategoryMapper {
+    private final SubcategoryMapper subcategoryMapper;
+
+    public Category categoryVoToNewCategory(CategoryVo categoryVo) {
+        Category newCategory = new Category();
+        newCategory.setName(categoryVo.getName());
+        List<Subcategory> newSubcategories = categoryVo.getSubcategories().stream()
+                .map(subcategoryDto -> subcategoryMapper.subcategoryVoToNewSubcategory(subcategoryDto, newCategory))
```

Comment:
Simplify method name - by using `subcategoryMapper` you already shown that here will be remapping, so method name must be readable better.
You also could change how it works like `subcategoryToSubcategoryVo`.